store/models.py

```diff
@@ -64,12 +70,18 @@ class Order(models.Model):
 
 
 class OrderItem(models.Model):
+    # TODO: Why do we want to keep orderitem if an order is deleted?
```

Comment:
Let's say you have to delete an order (for any reason) the order items are returned to the cart so that you can repurchase them if need be.  But now that i think about it, not sure it makes sense when i say it out loud